changes/5095.enhance.md

```diff
@@ -0,0 +1 @@
+Separate repository layer from container registry service
```

Comment:
```suggestion
Separate admin repository layer from container registry service
```

Reply:
I think the current message is appropriate because the important thing is not separating the admin, but separating the repository layer from the service.

Reply:
Isn't the container repository already separated, and isn't this PR about separating the admin container repository layer?

Reply:
I didn't consider it separated because, although there was a repository class, the actual implementation wasn't separated.